opwen_email_server/utils/email_parser.py

```diff
@@ -225,6 +226,10 @@ def format_inline_images(email: dict, on_error: Callable) -> dict:
     return new_email
 
 
+def descending_timestamp(email_sent_at: str) -> int:
```

Comment:
Given that we'll always use the return value of this function as a string, I'd recommend to make this explicit by returning a string here. This will also make our testing more representative since we'll ensure that we're testing on lexicographical order as opposed to numeric order.